palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/ExpressionSwitch.output

```diff
@@ -0,0 +1,21 @@
+class ExpressionSwitch {
+    boolean odd(int x) {
+        return switch (x) {
+            case 0 -> true;
+            case 1 -> false;
+            default -> {
+                int y = x - 1;
+                return odd(y);
+            }
+        };
+    }
+
+    int yield() {
+        return switch (i) {
+            case 0:
+                yield 0;
+            default:
+                yield n / i;
```

Comment:
woah this is a funky keyword... based on the reading of https://www.baeldung.com/java-thread-yield it sounds like not something that we expect palantir codebases to contain!